src/react/components/pages/editorPage/editorTagsInput.test.tsx

```diff
@@ -117,92 +133,34 @@ describe("Tags Input Component", () => {
         expect(wrapper.find(EditorTagsInput).state().tags).toHaveLength(originalTags.length);
     });
 
-    it("ctrl clicking tag opens editor modal", () => {
-        const onChangeHandler = jest.fn();
-        const wrapper = createComponent({
-            tags: originalTags,
-            displayHotKeys: true,
-            onChange: onChangeHandler,
-        });
-        expect(wrapper.find(EditorTagsInput).state().showModal).toBe(false);
-        wrapper.find("div.tag")
-            .first()
-            .simulate("click", { target: { innerText: originalTags[0].name}, ctrlKey: true});
-        expect(wrapper.find(EditorTagsInput).state().showModal).toBe(true);
-    });
-
-    it("ctrl clicking tag sets selected tag", () => {
-        const onChangeHandler = jest.fn();
-        const wrapper = createComponent({
-            tags: originalTags,
-            displayHotKeys: true,
-            onChange: onChangeHandler,
-        });
-        wrapper.find("div.tag")
-            .first()
-            .simulate("click", { target: { innerText: originalTags[0].name}, ctrlKey: true});
-        expect(wrapper.find(EditorTagsInput).state().selectedTag.id).toEqual(originalTags[0].name);
-        expect(wrapper.find(EditorTagsInput).state().selectedTag.color).toEqual(originalTags[0].color);
-    });
-
     it("ctrl clicking tag does not call onTagClick or OnTagShiftClick", () => {
         const onChangeHandler = jest.fn();
         const onTagClickHandler = jest.fn();
         const onTagShiftClickHandler = jest.fn();
+        const onTagCtrlClickHandler = jest.fn();
+
         const wrapper = createComponent({
             tags: originalTags,
-            displayHotKeys: true,
             onChange: onChangeHandler,
-            onTagShiftClick: onTagShiftClickHandler,
+            onShiftTagClick: onTagShiftClickHandler,
             onTagClick: onTagClickHandler,
+            onCtrlTagClick: onTagCtrlClickHandler,
         });
         wrapper.find("div.tag")
             .first()
             .simulate("click", { target: { innerText: originalTags[0].name}, ctrlKey: true});
-        // Shows modal
-        expect(wrapper.find(EditorTagsInput).state().showModal).toBe(true);
+        expect(onTagCtrlClickHandler).toBeCalled();
         // Does not Call onTagShiftClick
         expect(onTagShiftClickHandler).not.toBeCalled();
         // Does not call onTagClick
         expect(onTagClickHandler).not.toBeCalled();
     });
 
-    it("clicking 'ok' in modal closes and calls onChangeHandler", () => {
```

Comment:
Seems like more valuable tests to keep...

Reply:
same as above. this component does not control the modal.